### DISCUSSION TITLE
Feature request: Footnotes

### DISCUSSION THREAD

**Discussion by NoodleNodes** [notaapp/talk#283] [Feature Requests]
Hi team, 

Hopefully a relatively simple request, rendered **footnotes** would be a wonderful addition, especially for professional writers, and is largely available in competing Markdown editors. 

Please see [Extended Syntax | Markdown Guide](https://www.markdownguide.org/extended-syntax/#syntax-highlighting) for the implementation. 

As always, thank you for your continued work 😊

**Comment by erusev** [comment]
Hi and sorry for taking so long to reply.

> rendered footnotes would be a wonderful addition

Do you mean using superscript for the footnotes, like this:

<img width="112" alt="image" src="https://user-images.githubusercontent.com/184170/185310287-67abc8dc-064a-4ea1-90d2-d7691a912583.png">

> see Extended Syntax | Markdown Guide for the implementation

We already support this syntax for footnotes, but it's not very obvious because we don't highlight them very well (we use almost the same color and it's hard to distinguish)

**Comment by NoodleNodes** [comment]
> We already support this syntax for footnotes, but it's not very obvious because we don't highlight them very well (we use almost the same color and it's hard to distinguish)

Thanks for getting back to me. That's great that footnotes are already implemented in rendered view, but would it be possible to make them 'active' like [[links]] are in the edit view. It would be awesome to be able to click and be immediately taken to the relevant footnote, or select with cursor position on a footnote and have a pop-over window display the related footnote 😊🙏

**Comment by erusev** [comment]
> It would be awesome to be able to click and be immediately taken to the relevant footnote, or select with cursor position on a footnote and have a pop-over window display the related footnote 😊🙏

Sure, makes a lot of sense, I'll see what we can do.
